pyproject.toml

```diff
@@ -13,6 +13,9 @@ click = "*"
 requests = "*"
 shortuuid = "*"
 certifi = "*"
+"oslo.log" = "^5.0.2"
+"oslo.config" = "^9.0.0"
+"oslo.concurrency" = "^5.0.1"
```

Comment:
it is possible that magnum-capi will get installed in many different versions of OpenStack, which will have different locks on this.

Can you move this to `*` so that it will work as long as the dependency is there.

I doubt the API of those will be changing anytime soon.